src/init-ceph.in

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
```

Comment:
@joscollin 
'mmmm, I guess something went wrong with the PR submit
Will check, thanx.